### PR TITLE
Remove verifier log.info from test

### PR DESCRIPTION
**Pull request by suqinhuang**
It has been descriped in framework

Signed-off-by: Suqin Huang <shuang@redhat.com>

### CODE REVIEW

**Comment by suqinhuang** [comment]
https://github.com/avocado-framework/avocado-vt/pull/559

**Comment by pingl2016** [comment]
Shall we need to remove the below logging sentence in script iozone_windows.py and block_resize.py?
error_context.context("Enable %s driver verifier" % driver_name, logging.info)

**Comment by suqinhuang** [comment]
@pingl2016 have updated, could you review again

Thanks

File: qemu/tests/balloon_service.py [hunk 1]
```diff
@@ -165,8 +162,6 @@ def balloon_memory(session, device_path):
         uninstall_balloon_service = params["uninstall_balloon_service"] % drive_letter
         session.cmd(uninstall_balloon_service, ignore_all_errors=True)
 
-        error_context.context("Clear balloon driver verifier in guest",
-                              logging.info)
         if params.get("need_clear_verifier", "yes") == "yes":
```
Comment (pingl2016) [inline qemu/tests/balloon_service.py]: I think "need_clear_verifier" also can be removed as it is always "yes"

File: qemu/tests/balloon_stress.py [hunk 2]
```diff
@@ -80,8 +77,6 @@ def run(test, params, env):
     vm.verify_alive()
 
     if params.get("need_clear_verifier", "yes") == "yes":
```
Comment (pingl2016) [inline qemu/tests/balloon_stress.py]: same issue as balloon_service.py

**Comment by pingl2016** [comment]
Acked-by: Ping Li pingl@redhat.com

File: qemu/tests/balloon_service.py [hunk 3]
```diff
@@ -75,25 +75,29 @@ def memory_check(vm, get_polling_output, keyname):
...
+
+        error_context.context("Compare memory from guest with qmp",
+                              logging.info)
         if (abs(guest_mem - stat_memory_qmp)) > (guest_mem * check_mem_ratio):
-            raise exceptions.TestFail("%s of guest %s is not equal to %s in"
-                                      " qmp, the ratio is %s" % (keyname,
-                                                                 guest_mem,
-                                                                 stat_memory_qmp,
-                                                                 check_mem_ratio))
+            raise exceptions.TestFail("%s from guest: %s\n"
```
Comment (xutian) [inline qemu/tests/balloon_service.py]: I like original it looks more clean the new one.

File: qemu/tests/balloon_service.py [hunk 4]
```diff
@@ -142,34 +148,24 @@ def balloon_memory(session, device_path):
     drive_letter = get_disk_vol(session)
 
     try:
-        error_context.context("Enable %s driver verifier in guest" % driver_name,
-                              logging.info)
-        if params.get("need_enable_verifier", "yes") == "yes":
-            session = utils_test.qemu.setup_win_driver_verifier(session, driver_name,
-                                                                vm, timeout)
+        session = utils_test.qemu.setup_win_driver_verifier(session,
+                                                            driver_name, vm)
```
Comment (xutian) [inline qemu/tests/balloon_service.py]: now this function return None

File: qemu/tests/balloon_service.py [hunk 5]
```diff
@@ -142,34 +148,24 @@ def balloon_memory(session, device_path):
...
-        error_context.context("Clear balloon driver verifier in guest",
-                              logging.info)
-        if params.get("need_clear_verifier", "yes") == "yes":
-            session = utils_test.qemu.clear_win_driver_verifier(session, vm,
-                                                                timeout)
-        if session:
-            session.close()
+        uninstall_cmd = params["uninstall_balloon_service"] % drive_letter
+        session.cmd(uninstall_cmd, ignore_all_errors=True)
+        session = utils_test.qemu.clear_win_driver_verifier(session, vm)
```
Comment (xutian) [inline qemu/tests/balloon_service.py]: same as above line

File: qemu/tests/balloon_stress.py [hunk 6]
```diff
@@ -36,12 +36,9 @@ def run(test, params, env):
         session.cmd(params.get("pre_cmd"))
 
     driver_name = params["driver_name"]
-    if params.get("need_enable_verifier", "yes") == "yes":
-        error.context("Enable %s driver verifier in guest" % driver_name,
-                      logging.info)
-        session = utils_test.qemu.setup_win_driver_verifier(session,
-                                                            driver_name,
-                                                            vm, timeout)
+    session = utils_test.qemu.setup_win_driver_verifier(session,
```
Comment (xutian) [inline qemu/tests/balloon_stress.py]: same as above

File: qemu/tests/balloon_stress.py [hunk 7]
```diff
@@ -78,10 +75,6 @@ def run(test, params, env):
     error.context("verify guest still alive", logging.info)
     session.cmd(params["stop_player_cmd"])
     vm.verify_alive()
```
Comment (xutian) [inline qemu/tests/balloon_stress.py]: same as above

File: qemu/tests/block_resize.py [hunk 8]
```diff
@@ -59,17 +59,14 @@ def compare_block_size(session, block_cmd, block_pattern):
...
-        try:
-            session = utils_test.qemu.setup_win_driver_verifier(
-                      session, params["driver_name"], vm, timeout)
-            funcatexit.register(env, params.get("type"),
-                                utils_test.qemu.clear_win_driver_verifier,
-                                session, vm, timeout)
-        except Exception, e:
-            raise error.TestFail(e)
+    try:
+        session = utils_test.qemu.setup_win_driver_verifier(
```
Comment (xutian) [inline qemu/tests/block_resize.py]: same as above

**Comment by xutian** [comment]
And utils_test.qemu.clear/setup_win_driver_verifier  no need to pass session param we have vm object, we can get session from vm object.

**Comment by PyLearner** [comment]
Acked-by : Meng Yang meyang@redhat.com

**Comment by xutian** [comment]
@PyLearner  do you really test with latest code?

File: qemu/tests/balloon_stress.py [hunk 9]
```diff
@@ -36,12 +36,8 @@ def run(test, params, env):
         session.cmd(params.get("pre_cmd"))
 
     driver_name = params["driver_name"]
-    if params.get("need_enable_verifier", "yes") == "yes":
-        error.context("Enable %s driver verifier in guest" % driver_name,
-                      logging.info)
-        session = utils_test.qemu.setup_win_driver_verifier(session,
-                                                            driver_name,
-                                                            vm, timeout)
+    utils_test.qemu.setup_win_driver_verifier(session, driver_name, vm)
```
Comment (pingl2016) [inline qemu/tests/balloon_stress.py]: Need to remove parameter session here, as it has been dropped in avocado-vt#600

File: qemu/tests/driver_in_use.py [hunk 10]
```diff
@@ -84,12 +84,9 @@ def run_subtest(sub_type):
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     error_context.context("Boot guest with %s device" % driver, logging.info)
-    session = vm.wait_for_login(timeout=timeout)
 
-    error_context.context("Enable %s driver verifier in guest" % driver,
-                          logging.info)
-    session = utils_test.qemu.setup_win_driver_verifier(session,
-                                                        driver, vm, timeout)
+    utils_test.qemu.setup_win_driver_verifier(session, driver, vm)
```
Comment (pingl2016) [inline qemu/tests/driver_in_use.py]: Need to remove parameter session here, as it has been dropped in avocado-vt#600.

**Comment by pingl2016** [comment]
I think the parameter session for setup_win_driver_verifier() in case win_virtio_driver_update_test should be dropped.

File: qemu/tests/balloon_stress.py [hunk 11]
```diff
@@ -78,10 +74,4 @@ def run(test, params, env):
...
     session.cmd(params["stop_player_cmd"])
     vm.verify_alive()
-
-    if params.get("need_clear_verifier", "yes") == "yes":
-        error.context("Clear %s driver verifier in guest" % driver_name,
-                      logging.info)
-        session = utils_test.qemu.clear_win_driver_verifier(session, vm, timeout)
-    if session:
-        session.close()
+    utils_test.qemu.clear_win_driver_verifier(session, driver_name, vm)
```
Comment (pingl2016) [inline qemu/tests/balloon_stress.py]: Need to remove parameter session here

File: qemu/tests/balloon_stress.py (hunk 9 above)
Comment (CongLi) [inline qemu/tests/balloon_stress.py]: Yes, please update it.

**Comment by pingl2016** [comment]
It's ok for me now.

**Comment by CongLi** [comment]
For linux guest, it should not enable verifier.
Test block_resize:

2016-07-22 15:07:58: [root@unused ~]# verifier /querysettings
2016-07-22 15:07:58: bash: verifier: command not found...
2016-07-22 15:07:58: [root@unused ~]#
2016-07-22 15:07:58: verifier /reset
2016-07-22 15:07:58: bash: verifier: command not found...
2016-07-22 15:07:58: [root@unused ~]#
2016-07-22 15:07:58: echo $?
2016-07-22 15:07:58: 127
2016-07-22 15:07:58: [root@unused ~]#
2016-07-22 15:07:58:

Although there wont raise error, due to ignore error here.
session.cmd(verifier_setup_cmd,
                        timeout=timeout,
                        ignore_all_errors=True)

**Comment by suqinhuang** [comment]
@CongLi updated, thanks

**Comment by suqinhuang** [comment]
PASS 1-smp_1.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.block_resize
 FAIL 2-smp_1.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.balloon_service -> TestFail: stat-free-memory of guest 49 is not equal to 60 in qmp, the acceptable ratio is 0.1

PASS 3-smp_1.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.balloon_stress

PASS 4-smp_1.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.iozone_windows.aio_native

File: qemu/tests/block_resize.py [hunk 12]
```diff
@@ -57,20 +56,12 @@ def compare_block_size(session, block_cmd, block_pattern):
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     timeout = float(params.get("login_timeout", 240))
-    session = vm.wait_for_login(timeout=timeout)
+    driver_name = params["driver_name"]
```
Comment (pingl2016) [inline qemu/tests/block_resize.py]: better to use params.get("driver_name"), as i cannot found driver_name for linux guest in configure file.

**Comment by pingl2016** [comment]
Acked-by: Ping Li pingl@redhat.com

**Pull request merged**